frontend/src/metabase/meta/types/Card.js

```diff
@@ -10,16 +10,20 @@ export type UnsavedCard = {
     dataset_query: DatasetQuery,
     display: string,
     visualization_settings: VisualizationSettings,
-    parameters?: Array<Parameter>
+    parameters?: Array<Parameter>,
+    original_card_id?: CardId
```

Comment:
I don't think we should be including `original_card_id` here. The types generally should match what's returned by the backend.